src/containers/transfer/components/EthTransferForm.js

```diff
@@ -0,0 +1,164 @@
+import React from "react";
+import { connect } from "react-redux";
+import { reduxForm, SubmissionError, Field } from "redux-form";
+import store from "modules/store";
+import { EthSubmissionErrorPanel, EthSubmissionSuccessPanel } from "components/MsgPanels";
+import Button from "components/augmint-ui/button";
+import { Form, Validations, Normalizations } from "components/BaseComponents";
+import { transferEth, ETH_TRANSFER_SUCCESS } from "modules/reducers/ethTransfer";
+import theme from "styles/theme";
+
+class EthTransferForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.initialVal = this.props.initialValues.ethAmount;
+        this.state = {
+            result: null,
+            amount: this.initialVal
+        };
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.onEthAmountChange = this.onEthAmountChange.bind(this);
+        this.toggleEthTransferForm = this.toggleEthTransferForm.bind(this);
+    }
+
+    toggleEthTransferForm(e) {
+        this.props.toggleEthTransferForm(e);
+    }
+
+    onEthAmountChange(e) {
+        let amount;
+        try {
+            amount = parseFloat(e.target.value);
+            if (isNaN(amount)) {
+                amount = 0;
+            } else {
+                amount = parseFloat(e.target.value);
+            }
+        } catch (error) {
```

Comment:
So is `parseFloat` supposed to return NaN or throw on invalid input? It should be one or the other, so drop the unused path.